releases/v20.2.11.md

```diff
@@ -55,6 +55,7 @@ $ docker pull cockroachdb/cockroach-unstable:v20.2.11
 ### Bug fixes
 
 - [`BACKUP`](../v20.2/backup.html) no longer resolves intents one by one. This eliminates the need to run a high-priority query to clean up intents to unblock `BACKUP` in case of intent buildup. [#64931][#64931]
+- Fixed a bug that could cause backups to be slower. `LockTableWaiter` now checks the finalized transaction cache before pushing transactions to avoid duplicating work. [#64999][#64999]
```

Comment:
@mikeCRL This is the bug we were talking about earlier. Both this one and the top bug fix here (64931) are related to solving that original issue (59704). I assumed we would want to keep both here, but I'm unsure?

Reply:
The new text is good and keeping both sounds good to me. Thanks.